### ISSUE TITLE
Could you share the pretrained EmbeddingNet in "utils/cal_metrics.py"? Thanks.

### ISSUE THREAD

**Issue by bob35buaa** [JeremyCJM/DiffSHEG#14]
In your paper, you use FMD, FGD, and FED to evaluate the Fr ́ echt distance between generated and real data in gesture feature space. I want to follow your work and compute these metrics to evaluate my method. Could you share the pretrained EmbeddingNet in "[utils/cal_metrics.py](https://github.com/JeremyCJM/DiffSHEG/blob/3ebf3058f48cba3da9146afb7623e9ec1ab9e9a5/utils/cal_metrics.py#L460)"? Thanks.

**Comment by JeremyCJM** [comment]
Hi bob, the pretrained autoencoder weights can be downloaded here: https://drive.google.com/file/d/1Wm2WMlacwStFaciCh7UlhQeyA3E2yEnj/view?usp=sharing . The evaluation code can be directly found in *trainer.py. cal_metrics.py is not used in our repo.

**Comment by bob35buaa** [comment]
> Hi bob, the pretrained autoencoder weights can be downloaded here: https://drive.google.com/file/d/1Wm2WMlacwStFaciCh7UlhQeyA3E2yEnj/view?usp=sharing . The evaluation code can be directly found in *trainer.py. cal_metrics.py is not used in our repo.

Thank you very much!

**Comment by bob35buaa** [comment]
> Hi bob, the pretrained autoencoder weights can be downloaded here: https://drive.google.com/file/d/1Wm2WMlacwStFaciCh7UlhQeyA3E2yEnj/view?usp=sharing . The evaluation code can be directly found in *trainer.py. cal_metrics.py is not used in our repo.

Hi Jeremy. I also want to ask you about the training processing for FGD. I plan to train the autoencoder on my own dataset (SMPL-X format) for another task. Can you share the training scripts or the hyper-parameters about training, such as optimizer and learning rate?
